util/src/main/java/io/kubernetes/client/util/ClientBuilder.java

```diff
@@ -254,8 +260,11 @@ public static ClientBuilder cluster() throws IOException {
   }
 
   protected ClientBuilder setBasePath(String host, String port) {
+    if (host == null) {
```

Comment:
```suggestion
    if (StringUtils.isEmpty(host))
```
to handle the empty string, wdyt?